singlestore/website/singlestore.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kubedb.com/v1alpha2
+kind: Singlestore
+metadata:
+  name: sdb-sample
```

Comment:
`sdb` or `singlestore` is better.  Also chnage the namespace of pgppol to `demo`